README.md

```diff
@@ -436,14 +446,14 @@ USAGE
   trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
 
 OPTIONS
-  -c, --products-file-csv=products-file-csv                                         [default: ~/.commerce
-                                                                                    /examples/csv/Alpine-small.csv] The
+  -c, --products-file-csv=products-file-csv                                         [default: ~/.commerc
+                                                                                    e/examples/csv/Alpine-small.csv] The
```

Comment:
nit: format

Reply:
This file is automatically updated, the format always look weird in the diff but the README itself should be fine. Honestly for future work, I can edit the husky file such that we don't have the readme and package.json generated every time.